package.json

```diff
@@ -15,7 +15,7 @@
     "lodash": "^4.4.0"
   },
   "devDependencies": {
-    "async": "^1.4.0",
+    "bluebird": "^3.4.1",
```

Comment:
What do you think about moving to fixed versions (without `^`)? It makes `david` work much better, and causes less breakage from packages not following semver.